prod-devel-rcar.yaml

```diff
@@ -547,9 +547,14 @@ parameters:
               - type: git
                 url: "ssh://git@gitpct.epam.com/epmd-aepr/meta-aos"
                 rev: master
+              - type: git
+                url: "https://github.com/xen-troops/meta-xt-prod-devel-rcar.git"
+                rev: master
             builder:
               layers:
                 - " ../meta-aos"
+                - "../meta-xt-prod-devel-rcar/meta-aos-tune"
+                - "../meta-xt-prod-devel-rcar/meta-xt-prod-devel-rcar-driver-domain"
```

Comment:
IMO, `meta-xt-prod-devel-rcar-driver-domain` should be included non-conditionally. This will prevent confusion in the future, when someone will try to add non AOS-related receipt.